godtossing/Pos/Ssc/GodTossing/Core/Types.hs

```diff
@@ -244,11 +248,31 @@ getCertId = addressHash . vcSigningKey
 
 -- | VssCertificatesMap contains all valid certificates collected
 -- during some period of time.
-type VssCertificatesMap = HashMap StakeholderId VssCertificate
+newtype VssCertificatesMap = VssCertificatesMap
```

Comment:
Our convention is to either not export constructor or call it with `Unsafe` prefix when it has some internal invariants. In this particular case we want `addressHash (vcKey v) == k` for each `(k, v)` in map.